src/lhttpc.erl

```diff
@@ -32,18 +32,22 @@
 -module(lhttpc).
 -behaviour(application).
 
--export([start/0, stop/0, request/4, request/5, request/6, request/9]).
--export([start/2, stop/1]).
--export([
-        send_body_part/2,
-        send_body_part/3,
-        send_trailers/2,
-        send_trailers/3
-    ]).
--export([
-        get_body_part/1,
-        get_body_part/2
-        ]).
+%% this is the API module; xref ignore all exports
+-ignore_xref(
+   [start/0, stop/0, start/2, stop/1
+    , lb_status/0
+    , request/4, request/5, request/6, request/9
+    , send_body_part/2, send_body_part/3
+    , send_trailers/2, send_trailers/3
+    , get_body_part/1, get_body_part/2]).
+
+-export(
+   [start/0, stop/0, start/2, stop/1
+    , lb_status/0
+    , request/4, request/5, request/6, request/9
+    , send_body_part/2, send_body_part/3
+    , send_trailers/2, send_trailers/3
+    , get_body_part/1, get_body_part/2]).
```

Comment:
Looking at it, lhttp_sock, lhttpc_lb, lhttpc_lib, and lhttpc all followed the same style of having trailing commas rather than commas at the beginning of lines. What files were conflicting?